src/mobu/constants.py

```diff
@@ -3,16 +3,31 @@
 from __future__ import annotations
 
 from datetime import timedelta
+from pathlib import Path
 
 __all__ = [
+    "GITHUB_CI_SCOPES",
+    "GITHUB_REPO_CONFIG_PATH",
     "GITHUB_WEBHOOK_WAIT_SECONDS",
-    "NOTEBOOK_REPO_URL",
     "NOTEBOOK_REPO_BRANCH",
+    "NOTEBOOK_REPO_URL",
     "TOKEN_LIFETIME",
     "USERNAME_REGEX",
     "WEBSOCKET_OPEN_TIMEOUT",
 ]
 
+
+GITHUB_CI_SCOPES = [
+    "exec:notebook",
+    "exec:portal",
+    "read:image",
+    "read:tap",
+]
+"""All NotebookRunner business run via GitHub CI get these scopes."""
```

Comment:
@rra , any objections to giving these scopes to all of the mobu notebook runner jobs kicked from GitHub CI?

Reply:
Yep, making this configurable is a great idea.  This PR already adds another config file from a config map for the CI functionality, I can add it there in another PR.

Reply:
https://github.com/lsst-sqre/mobu/pull/354